storage/rocksdb/ha_rocksdb.cc

```diff
@@ -2436,7 +2436,6 @@ class Rdb_transaction {
       entire transaction.
     */
     do_set_savepoint();
-    m_n_savepoints= 1;
```

Comment:
Since this is not longer used, the definition in Rdb_transaction should also be removed.